Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -893,8 +905,13 @@ def update_outbound_command(args, params):
                                     category_default, category_attribute, collapse_ips)
 
     indicators = refresh_outbound_context(request_args)
-    hr = tableToMarkdown('List was updated successfully with the following values', indicators,
-                         ['Indicators']) if print_indicators == 'true' else 'List was updated successfully'
+    if len(indicators) > 0:
```

Comment:
It seems from `refresh_outbound_context` that the returned dict will never be of length 0, are you sure thats the correct way to do that?

Reply:
in case no indicators are fetched (lets say for example that the query did not return any results or the offset given is larger that the indicator list length) - then refresh_outbound_context would return an empty string.

Here I separate the two possibilities - I'll make the "not" change here as well